tic_tac_toe/src/board.py

```diff
@@ -0,0 +1,203 @@
+# Board Representation
+
+# test_board = [0, 1, -1, 0, 1, -1, 0, 1, -1]
+# 9 values, for 9 spaces on the board
+#
+# -1 = X
+# 0 = No Value
+# 1 = O
+#
+
+
+class Board:
+    def __init__(self, dim):
+        self.values = [None, None, None, None, None, None, None, None, None]
+
+        self.template = "{}|{}|{}\n-------\n{}|{}|{}\n-------\n{}|{}|{}"
+
+        self.dim = dim
+
+        self.winner = None
+
+    def convert(self, x, y):
+        """
+        converts a 2-dimensional coordinate to a 1-dimensional coordinate
```

Comment:
`x` and `y` doesn't give me much here.  What does the grid layout look like? Where is `(0,0)` and where is `(2,2)` on a 3x3 grid?

Reply:
In the comments, please write out the coordinates of the grid and their positions relative to one another for a 3x3 grid.

Reply:
On a 3x3 grid, the board would look like:

X|X|X 1
--------
X|X|X 2
--------
X|X|X 3
1 2 3

(1, 1) = 0
(1, 2) = 1
(1, 3) = 2

(2, 1) = 3
(2, 2) = 4
(2, 3) = 5

(3, 1) = 6
(3, 2) = 7
(3, 3) = 8

Reply:
Ok, if you want to go with 1-index, then you're going to have to change some logic.  I would recommend using 0-index (i.e. `(0,0)`, etc.)

Reply:
Ok, I think we figured this out.  You are going to go for a 0-based approach and translate to 1-based in the calling function.